<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameTable</class>
 <widget class="QMainWindow" name="GameTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KoolTruco</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="player2GroupBox">
        <property name="minimumSize">
         <size>
          <width>275</width>
          <height>170</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Player 2</string>
        </property>
        <widget class="QLabel" name="player2Label_card0a">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>28</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
         <property name="pixmap">
          <pixmap resource="kooltruco.qrc">:/deck/pia/back.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="player2Label_card0b">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>28</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
         <property name="pixmap">
          <pixmap resource="kooltruco.qrc">:/deck/pia/back.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="player2Label_card0c">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>28</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
         <property name="pixmap">
          <pixmap resource="kooltruco.qrc">:/deck/pia/back.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="player2PicLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>80</width>
           <height>60</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <zorder>player2Label_card0a</zorder>
        <zorder>player2PicLabel</zorder>
        <zorder>player2Label_card0b</zorder>
        <zorder>player2Label_card0c</zorder>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget_Left">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_Chat">
         <attribute name="title">
          <string>Chat</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="2" column="0">
           <widget class="QLineEdit" name="chatLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QTextBrowser" name="chatTextBrowser">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_Log">
         <attribute name="title">
          <string>Log</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTextBrowser" name="LogTextBrowser"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QPushButton" name="envidoPushButton">
            <property name="text">
             <string>Envido</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="florPushButton">
            <property name="text">
             <string>Flor</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="trucoPushButton">
            <property name="text">
             <string>Truco</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="giveupPushButton">
            <property name="text">
             <string>Correr</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="player3GroupBox">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>272</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Player 3</string>
          </property>
          <widget class="QLabel" name="player3Label_card0c">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>128</width>
             <height>80</height>
            </rect>
           </property>
           <property name="pixmap">
            <pixmap resource="kooltruco.qrc">:/deck/pia/back2.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="player3Label_card0b">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>128</width>
             <height>80</height>
            </rect>
           </property>
           <property name="pixmap">
            <pixmap resource="kooltruco.qrc">:/deck/pia/back2.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="player3Label_card0a">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>128</width>
             <height>80</height>
            </rect>
           </property>
           <property name="pixmap">
            <pixmap resource="kooltruco.qrc">:/deck/pia/back2.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="player3PicLabel">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>180</y>
             <width>60</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>80</height>
            </size>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <zorder>player3Label_card0a</zorder>
          <zorder>player3PicLabel</zorder>
          <zorder>player3Label_card0b</zorder>
          <zorder>player3Label_card0c</zorder>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="minimumSize">
         <size>
          <width>428</width>
          <height>398</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>428</width>
          <height>398</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <widget class="QLabel" name="p0Label_card1a">
         <property name="geometry">
          <rect>
           <x>132</x>
           <y>260</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p0Label_card1b">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>260</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p0Label_card1c">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>260</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p1Label_card1a">
         <property name="geometry">
          <rect>
           <x>295</x>
           <y>200</y>
           <width>128</width>
           <height>80</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p1Label_card1b">
         <property name="geometry">
          <rect>
           <x>295</x>
           <y>150</y>
           <width>128</width>
           <height>80</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p1Label_card1c">
         <property name="geometry">
          <rect>
           <x>295</x>
           <y>110</y>
           <width>128</width>
           <height>80</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p2Label_card1a">
         <property name="geometry">
          <rect>
           <x>212</x>
           <y>10</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p2Label_card1c">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p2Label_card1b">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>10</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p3Label_card1c">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>190</y>
           <width>128</width>
           <height>80</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p3Label_card1a">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>110</y>
           <width>128</width>
           <height>80</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="p3Label_card1b">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>150</y>
           <width>128</width>
           <height>80</height>
          </rect>
         </property>
        </widget>
        <zorder>p1Label_card1a</zorder>
        <zorder>p2Label_card1a</zorder>
        <zorder>p3Label_card1a</zorder>
        <zorder>p0Label_card1a</zorder>
        <zorder>p0Label_card1b</zorder>
        <zorder>p0Label_card1c</zorder>
        <zorder>p3Label_card1b</zorder>
        <zorder>p3Label_card1c</zorder>
        <zorder>p2Label_card1b</zorder>
        <zorder>p2Label_card1c</zorder>
        <zorder>p1Label_card1b</zorder>
        <zorder>p1Label_card1c</zorder>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="player1GroupBox">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>272</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Player 1</string>
          </property>
          <widget class="QLabel" name="player1Label_card0c">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>120</y>
             <width>128</width>
             <height>80</height>
            </rect>
           </property>
           <property name="pixmap">
            <pixmap resource="kooltruco.qrc">:/deck/pia/back2.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="player1Label_card0a">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>180</y>
             <width>128</width>
             <height>80</height>
            </rect>
           </property>
           <property name="pixmap">
            <pixmap resource="kooltruco.qrc">:/deck/pia/back2.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="player1Label_card0b">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>128</width>
             <height>80</height>
            </rect>
           </property>
           <property name="pixmap">
            <pixmap resource="kooltruco.qrc">:/deck/pia/back2.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="player1PicLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>60</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>80</height>
            </size>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <zorder>player1Label_card0a</zorder>
          <zorder>player1PicLabel</zorder>
          <zorder>player1Label_card0b</zorder>
          <zorder>player1Label_card0c</zorder>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="playerGroupBox">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>170</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Player</string>
        </property>
        <widget class="QPushButton" name="playerButton_card0">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>28</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>128</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>128</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="playerButton_card1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>28</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>128</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>128</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="playerButton_card2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>28</y>
           <width>80</width>
           <height>128</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>128</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>128</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="playerPicLabel">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>20</y>
           <width>60</width>
           <height>60</height>
          </rect>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="locale">
     <locale language="Portuguese" country="Brazil"/>
    </property>
    <property name="title">
     <string>&amp;Arquivo</string>
    </property>
    <addaction name="actionRestart"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="locale">
     <locale language="Portuguese" country="Brazil"/>
    </property>
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionRestart">
   <property name="text">
    <string>&amp;Reiniciar</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Sair</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;Sobre</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>Sobre &amp;Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="kooltruco.qrc"/>
 </resources>
 <connections/>
</ui>
